flux_led/__main__.py

```diff
@@ -714,28 +728,24 @@ def update_state(self, retry=2):
         #     |  |  |  |  |  speed: 0x01 = highest 0x1f is lowest
         #     |  |  |  |  Mode WW(01), WW+CW(02), RGB(03), RGBW(04), RGBWW(05)
         #     |  |  |  preset pattern
-        #     |  |  off(23)/on(24)
+        #     |  |  off(23)/on(24) [bulbs], off(36)/on(35) [switches]
```

Comment:
This is in Hex.  Can you double-check two things.

1. Is it returning 36/35 in Hex?
2. Double check that 36 is off and 35 is on. It's reversed from everything else they do. I wouldn't be surprised but, we should be sure.

Reply:
My comments is in decimal.

```
        if power_state == 0x23:
            self._is_on = True
        elif power_state == 0x24:
            self._is_on = False
```

The existing inline comment is reverse.  The existing code below is correct.

I will fix the comment

`0x23` = `on`
`0x24` = `off`

Reply:
[`cdc37eb` (#83)](https://github.com/Danielhiversen/flux_led/pull/83/commits/cdc37eb14e6ebf5d8276f86b06b187d828353a94)